ex9/js/main.js

```diff
@@ -0,0 +1,95 @@
+require(['ajaxGet'], function(ajaxGet) {
+  $(function() {
+    var on;
+    setEvents();
+    on = {
+      loadList: function() {
```

Comment:
This file is WAY too big and deal with three different things. Please split these into three files:
1. searchui
1. sortui
1. data (which will later also include the code to do the searching/sorting).

This will also make you think about what values you need from #1 and #2, and will make them more like objects.